test/blackbox-tests/test-cases/reporting-of-cycles.t/run.t

```diff
@@ -67,3 +67,27 @@ cryptic and can involve unrelated files:
   -> required by _build/default/indirect/a.exe
   -> required by alias indirect/indirect-deps in indirect/dune:6
   [1]
+
+This is a reproduction case from issue #4345
+  $ DIR="gh4345"
+  $ mkdir $DIR && cd $DIR
+  $ echo "(lang dune 2.8)" > dune-project
+  $ mkdir lib
+  $ touch lib.opam file lib/lib.ml
+  $ cat >lib/dune <<EOF
+  > (library (name lib) (public_name lib))
+  > (copy_files (files ../file))
+  > EOF
+  $ dune build --root .
+  Error: Dependency cycle between:
+     Loading rules for directory _build/default
+  -> evaluating predicate
+     { dir = In_build_dir "default"
```

Comment:
This record is not very user friendly. `human_readable_description` is supposed to print something any user can understand.

Reply:
Changed it to print only the directory.